pages/android/input.md

```diff
@@ -2,6 +2,7 @@
 
 > Send event codes or touchscreen gestures to an Android device.
 > Event codes can be inputting single characters, ending calls, volume up, etc.
+> This command can only be used through `adb shell`.
```

Comment:
As I said [here](https://github.com/tldr-pages/tldr/pull/5808/files#r617509472), I'm not sure if it's ok to have a total of >3 description lines.  We can ask someone though, ping @sbrl @waldyrious @agnivade

Reply:
Interesting, I would have expected the linker to have a check for this. Here's what our [guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format) say about the main description:

> Preferably one line; two are acceptable if necessary.

It seems pretty clear that the max should be two lines.

Reply:
I've opened tldr-pages/tldr-lint#77 to add this check to the linter.

Reply:
What Waldir said.